--- origsrc/qutim-0.3.1/core/cmake/QtBundleUtils.cmake	2012-04-13 11:51:50.000000000 -0500
+++ src/qutim-0.3.1/core/cmake/QtBundleUtils.cmake	2013-11-14 03:32:02.376414300 -0600
@@ -89,7 +89,7 @@ macro(DEFINE_BUNDLE_PATHS _name)
                 set(BUNDLE_PATH "${CMAKE_INSTALL_PREFIX}/${BINDIR}/${BUNDLE_NAME}")
 		set(LIBDIR lib${LIB_SUFFIX})
                 set(RLIBDIR ${LIBDIR})
-                set(SHAREDIR share/apps/${_name})
+                set(SHAREDIR share/${_name})
                 set(PLUGINSDIR ${LIBDIR}/plugins/)
                 set(IMPORTSDIR ${LIBDIR}/imports)
 	endif()
--- origsrc/qutim-0.3.1/core/cmake/QutIMMacros.cmake	2012-04-13 11:51:50.000000000 -0500
+++ src/qutim-0.3.1/core/cmake/QutIMMacros.cmake	2013-11-14 03:32:15.096141800 -0600
@@ -94,7 +94,7 @@ IF(WIN32)
 	set(QUTIM_SHARE_DIR_DEF ${QUTIM_BUNDLE_NAME}/Contents/Resources/share)
 	set(SHARE_DIR_DEF ${QUTIM_SHARE_DIR_DEF})
 ELSE()
-	set(QUTIM_SHARE_DIR_DEF share/apps/qutim)
+	set(QUTIM_SHARE_DIR_DEF share/qutim)
 	set(SHARE_DIR_DEF share)
 ENDIF()
 set(QUTIM_SHARE_DIR "${QUTIM_SHARE_DIR_DEF}" CACHE STRING "QutIM share dir")
